docs/plugins/data.malware-scan.md

```diff
@@ -0,0 +1,26 @@
+data.malware-scan
+==========
+
+This plugin extracts hashs(md5,sha1,sha256) of each attachment and feeds them to MHR based blacklists such as 
+[MALWARE HASH REGISTRY][1].
+
+This plugin will reject any mail that have higher percentage equal to 25%.
+
+Configuration
+-------------
+
+This plugin reads configuration from data.malware-scan.ini.
+
+The main section defines global settings for all lists and the blacklists
+zones are specified as INI section headings with the configuration for 
+each zone within that section.
+
+The main section can contain the following options:
+
+* deny_percentage
+
+  Default: 25
+
+  The maximum percentage (return of MHR), any mail is rejected.
+
+[1]: http://www.team-cymru.org/MHR.html
```

Comment:
You should probably mention that the MHR is for low-volume non-commercial use - or quote the terms of usage here.